src/libultra/io/motor.c

```diff
@@ -0,0 +1,120 @@
+#include "ultra64.h"
+#include "global.h"
+
```

Comment:
Is this the correct name for this file? It's currently called `rumblepak.c`.

Reply:
libultra used motor in place of pak/rumble/mempak. the file deals with both the rumblepak and memory pak so idk maybe name the file pak.c ?